python/lsst/obs/decam/decamMapper.py

```diff
@@ -210,10 +213,12 @@ def std_raw(self, item, dataId):
         rawPath = self.map_raw(dataId).getLocations()[0]
         headerPath = re.sub(r'[\[](\d+)[\]]$', "[0]", rawPath)
         md0 = afwImage.readMetadata(headerPath)
-        # Keywords EXPTIME and MJD-OBS are used to set the calib object.
-        for kw in ('EXPTIME', 'MJD-OBS', 'DARKTIME'):
+        # extra keywords to copy to the exposure
+        for kw in ('DARKTIME', ):
```

Comment:
Why not include darktime in `VisitInfo`?  It's required for the dark correction, and it would be good not to have to grab it from the header in ISR.